deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -55,6 +55,18 @@ metadata:
                     "strategy": "persistent"
                   }
                 }
+              },
+              "logs": {
+                "loki": {
+                  "enabled": false,
+                  "flavor": "1.extra-small",
+                  "replicationFactor": 1,
+                  "storage": {
+                    "objectStorageSecret": "test",
+                    "storageClass": "standard"
+                    }
+                  }
+                }
```

Comment:
I think this is causing an issue. I just noticed that there isn't a corresponding change to the `deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml` is missing here. Normally when you build the local bundle it should take the `CR` example file and will generate the example in the generated bundle CRD.

There needs to be a command run by `operator-sdk` that I'll look in to here. It's been a while since I've done it. May have a change incoming before we can merge this since CI is failing.

Reply:
Yea confirmed. Changes that show up in the bundle also need to be put in to the `deploy/crds/` files, which are ultimately the source when you generate the bundle.

You'll want to run this command: `operator-sdk generate bundle --default-channel unstable --channels unstable` which will show you the changes being removed from the `deploy/olm-catalog/` because they aren't sourced in the `deploy/crds/` directory files.

Reply:
Here is a small patch for part of the changes because I was testing locally:

```diff
diff --git a/deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml b/deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml
index 7752eed..f84c060 100644
--- a/deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml
+++ b/deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml
@@ -34,6 +34,14 @@ spec:
           persistent:
             storageSelector: {}
             pvcStorageRequest: 20Gi
+    logs:
+      loki:
+        enabled: true,
+        flavor: 1.extra-small
+        replicationFactor: 1
+        storage:
+          objectStorageSecret: test
+          storageClass: standard
   graphing:
     enabled: false
     grafana:
```